src/initialization/InitDistribution.H

```diff
@@ -98,6 +98,26 @@ namespace impactx::initialization
         amrex::ParticleReal* const AMREX_RESTRICT m_part_py;
         amrex::ParticleReal* const AMREX_RESTRICT m_part_pt;
     };
+
+    /** Initialize the input parameters for all distributions that read phase space ellipse parameters from the input
+     *
```

Comment:
Please add `@param[in] name description here` doxygen strings on the two new function declarations here.